sdk/maps/maps-route-rest/samples-dev/resumeLro.ts

```diff
@@ -70,14 +70,13 @@ async function main(): Promise<void> {
     body: request,
   });
 
-  const initialPoller = getLongRunningPoller(client, initialResponse);
+  const initialPoller = await getLongRunningPoller(client, initialResponse);
   /* We can get a partial of the results first */
-  await initialPoller.poll();
   /** Serialized the current operation for future poller */
-  const serializedState = initialPoller.toString();
+  const serializedState = await (await initialPoller).serialize();
```

Comment:
why are two `await` here? I think we could do followings.
```ts
const serializedState = await initialPoller.serialize();

// or but marked as deprecated now so not recommanded
const serializedState = initialPoller.toString();
```

Reply:
Thanks! We have await getLongRunningPoller so second await is not needed indeed. :)